modules/pwa/index.ts

```diff
@@ -56,8 +56,10 @@ export default defineNuxtModule<VitePWANuxtOptions>({
     else {
       // Register PWA types
       nuxt.hook('prepare:types', ({ references }) => {
+        const types = resolver.resolve('runtime/types')
+        references.push({ path: resolver.resolve(nuxt.options.buildDir, types) })
```

Comment:
why the double resolve here? shouldn't this just be `resolver.resolve('runtime/types')`?

Reply:
upps

Reply:
this may be the problem, testing on local we don't need the type in the plugin (ofc, the module is in local, I'll review Nuxt PWA module)